handbook/rbac/rbac.md

```diff
@@ -0,0 +1,128 @@
+
+    <figure>
+        <img src="../../img/allow-all-privileges.png" width="500" />
+        <figcaption>All privileges for an area are granted with Allow all privileges is checked</figcaption>
+    </figure>
+
+#### Built-in and Custom Roles
+
+Roles define the privileges a user may take against resources within SystemLink. While a user has a role within the context of a workspace a role is defined globally. This enables the same role to be used across multiple workspaces.
+
+When creating an assigning roles, it is best practice to provide users with the minimum number of privileges to perform their work. SystemLink has granular privileges for services and application in the product. These privileges also include access to individual SystemLink applications such as **Systems Manager** and **Test Monitor**.
+
+!!! note
+    If a user has a role that grants access to an application in any workspace that application will be available to the user at all times. If a user has not been granted access to an application in any workspace that application will be hidden from the navigation menu.
+
+Custom roles may be created, duplicated, modified, and deleted as needed. If a role is updated all users mapped to that role are affected.
+
+Due to the large number of privileges available SystemLink includes several *built-in* roles to facilitate getting started. Go to the SystemLink **Security** application to review the details on privileges granted for each role.
+
+- **Systems Maintainer**: This role has full access to systems, packages, assets, alarms, dashboards, and WebVIs. This role had read-only access to data and analysis procedures.
+
+- **Data Maintainer**: This role has full access to data (files, tag, and test results), as well as DataFinder, Data Preparation, Analysis Automation, dashboards, and WebVIs. This role has read only access to systems, packages, and feeds.
+
+- **Collaborator**: This role is read-only for all resources within SystemLink.
+
+- **Automated Agent**: This role does not have access to any SystemLink applications. While it has privileges to create data, and some cases modify data, it cannot delete data. This role has no access to systems, package repository, states, or analysis routines.
+
+!!! note
+    Built in roles cannot be modified. They can be duplicated and the duplicated roles may be renamed and the privileges modified. This constraint exists because NI may add or change privileges for built-in roles between releases. Users assigned these built-in roles will have their privileges automatically change upon upgrade. To opt out of this behavior assign your users custom roles.
```

Comment:
```suggestion
    Built in roles cannot be modified. They can be duplicated and the duplicated roles may be renamed and the privileges modified. This constraint exists because NI may add or change privileges for built-in roles between releases. Users assigned these built-in roles will have their privileges automatically changed upon upgrade. To opt out of this behavior assign your users custom roles.
```